feature/metis/conversation/src/main/kotlin/de/tum/informatics/www1/artemis/native_app/feature/metis/conversation/ui/chatlist/MetisChatList.kt

```diff
@@ -96,6 +97,7 @@ internal fun MetisChatList(
         clientId = clientId,
         hasModerationRights = hasModerationRights,
         isAtLeastTutorInCourse = isAtLeastTutorInCourse,
+        isConversationCreator = isConversationCreator,
```

Comment:
At this point I feel like we maybe should wrap these flags in a wrapper class to reduce the clutter (as they are passed down a lot). But I cant really think of a suitable name, maybe something in the direction of `PostActionFlags`?